charts/ks-devops/values.yaml

```diff
@@ -24,7 +24,7 @@ fullnameOverride: ""
 
 serviceAccount:
   # Specifies whether a service account should be created
-  create: false
+  create: true
```

Comment:
IMO, keep this value original is great unless you tell us why we make it true as default.

Reply:
By default (create: false), serviceaccount is not created and ks-devops controllers cannot be used